mono/mini/mini-llvm.c

```diff
@@ -7440,6 +7440,9 @@ emit_method_inner (EmitContext *ctx)
 
 	mono_llvm_add_func_attr (method, LLVM_ATTR_UW_TABLE);
 
+	if (!cfg->llvm_only)
```

Comment:
Why do we need that?

_Feel free to discard the review whenever the question is answered, this is not requested changes but merely a question for documentation_

Reply:
I was seeing a lot of weird behavior in the corlib tests that suggested a few edge cases to consider. I think that enabling inlining for the majority of methods is a good idea, but that fitting that into this PR would make it hard to reason about.

In the past, we weren’t inlining any methods because we were obfuscating all of the call sites. This preserves that behavior.

Also not sure how letting LLVM do inlining impacts things like non-C++ EH. There are a number of correctness hurdles there.

Adding a comment to the source to that effect.

As for the reason to disable this on llvmonly, it's clear that this is currently working on llvmonly without it (because this is already the default on llvmonly). I didn't want to change any of the behavior in this PR besides the intended change.  I'm pretty sure that things should build on bitcode without that conditional.  It *shouldn't* result in any optimization losses, but I figured I'd wait until I took on inlining in general to look into that.